lambdas/service/app.py

```diff
@@ -782,6 +788,11 @@ def validate_filters(filters):
             else:
                 raise BadRequestError(f'The relation in the `filters` parameter entry for `{facet}`'
                                       f' must be one of {valid_relations}')
+            if relation == 'is':
+                for fields in value:
+                    if isinstance(fields, list):
+                        raise BadRequestError('The value of the `is` relation in the `filters` '
+                                              'parameter cannot be a list of lists.')
```

Comment:
I think it would be more durable if we threw an error _unless_ the fields have the _expected_ type. Unfortunately this is a little more work because it involves figuring out which types are expected.

Reply:
I was doing some related work on #3325  that helped identify which types are expected.

```
(.venv) CBSEspecter:azul amar$ jq '.paths."/index/files".get.parameters[1].content."application/json".schema.properties[] | .oneOf[0].is.properties.is.items.type' lambdas/service/openapi.json | sort | uniq
"boolean"
"integer"
"object"
"string"
```